R/step_epi_naomit.R

```diff
@@ -0,0 +1,20 @@
+#' Unified NA omission wrapper function for recipes
+#'
+#' @param recipe Recipe to be used for omission steps
+#'
+#' @return Omits NA's from both predictors and outcomes at training time
+#'   to fit the model. Also only omits associated predictors and not
+#'   outcomes at prediction time due to lack of response and avoidance
+#'   of data loss.
+#' @export
+#' @examples
+#' case_death_rate_subset %>%
+#'  epi_recipe() %>%
+#'  step_epi_naomit()
+
+step_epi_naomit <- function(recipe) {
+  stopifnot("recipe" %in% class(recipe))
```

Comment:
better is
```r
stopifnot(inherits(x, "recipe")
```

Reply:
I rerun `devtools::document` and the Rd files for create_layer and new_epi_recipe_blueprint haven't been deleted.
Also, I updated the second comment.